src/components/Card/Card.js

```diff
@@ -1,10 +1,10 @@
 import './Card.css'
 
-const Card = () => {
+const Card = ( { name, img }) => {
   return (
     <div className='card'>
-      <img src="https://ir.4sqi.net/img/general/original/3277_DPE4uVcVV57ISXxnBAtSV1n_2kgh2jFcCKt5X1ilPDA.jpg"/>
-      <h3>Hudson Hill</h3>
+      <img src={img}/>
+      <h3>{name}</h3>
       <h4>Rating: 100%</h4>
```

Comment:
Rating will be passed through as a prop too?

Reply:
Yes! We have to refactor that on the backend